packages/clerk-js/src/ui/utils/useFormControl.ts

```diff
@@ -85,13 +100,23 @@ export const useFormControl = <Id extends string>(
     setErrorText(translateError(error || undefined));
     if (typeof error !== 'undefined') {
       setIsSuccessful(false);
+      setWarningText('');
     }
   };
   const setSuccessful: FormControlState['setSuccessful'] = isSuccess => {
     setErrorText(undefined);
+    setWarningText('');
```

Comment:
❓ Why is warning text set to `""` but error text to `undefined`? I'd expect them to be aligned.

Reply:
@SokratisVidros This is something that was there initially, which is yeah I also followed that pattern when I started adding more state hooks.

I will align these so they only accept strings